lms/djangoapps/dashboard/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+
+from dashboard.models import EmailsAdressMailing
```

Comment:
@UvgenGen `EmailsAdressMailing` -> `EmailsAddressMailing` 
and in other places with `address` in vars' namings (e.g. `email_adress`)